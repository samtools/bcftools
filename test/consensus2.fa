>1
CTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTC
>2
CCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTC
>3
CCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTT
C
>4
CCCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGT
TC
