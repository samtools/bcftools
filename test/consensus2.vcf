##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
##ALT=<ID=DEL,Description="Deletion">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA001
1	59	.	C	a,Caa	.	PASS	.	GT	1|2
2	60	.	C	a,Caa	.	PASS	.	GT	1|2
3	61	.	C	a,Caa	.	PASS	.	GT	1|2
4	62	.	C	a,Caa	.	PASS	.	GT	1|2
