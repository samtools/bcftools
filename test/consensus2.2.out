>1
CTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTCA
A
>2
CCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTTC
AA
>3
CCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTT
CAA
>4
CCCCTACCATATGTGACATATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGT
TCAA
